### ISSUE TITLE
[BUG] Cannot manage search without login

### ISSUE THREAD

**Issue by Invictaz** [LivingWithHippos/unchained-android#302]
You need to login via authorize or api token to search for content, which is weird because you can also search for content to share it with other apps.

Also, the menu setting "bootup screen" which is set to search does not work because you are not logged in. There is alsno no warning for this.

**Comment by LivingWithHippos** [comment]
Yeah it's more like a feature but I guess I could code some way for the search Window to be used without logging in, just removing it's ID from the disable bottom item lists should work partially
